src/anomalib/engine/engine.py

```diff
@@ -536,30 +536,39 @@ def export(
             logger.exception(f"Unknown type {type(transform)} for transform.")
             raise TypeError
 
-        if export_path is None:
-            export_path = Path(self.trainer.default_root_dir)
+        if export_root is None:
+            export_root = Path(self.trainer.default_root_dir)
 
+        final_export_path: Path | None = None
```

Comment:
How about the following?
```suggestion
        exported_model_path: Path | None = None
```